### ISSUE TITLE
Add custom label for buttons

### ISSUE THREAD

**Issue by paologf** [coderaiser/smalltalk#14]
Hello,

is it possible add into the `options` parameter of `confirm` function a dictionary with a custom label for each button?
Maybe something like `{"cancel":"Cancel Button Label","close":"Cancel Button Label","ok":"Ok Button Label"}` , pass into the `getTemplate` function this `labels` dictionary and use call `labels[name] || name`.

```
   function getTemplate(title, msg, value, buttons, labels) {
    const encodedMsg = msg.replace(/\n/g, '<br>');
    
    return `<div class="page">
        <div data-name="js-close" class="close-button"></div>
        <header>${ title }</header>
        <div class="content-area">${ encodedMsg }${ value }</div>
        <div class="action-area">
            <div class="button-strip"> ${
                buttons.map((name, i) =>
                    `<button tabindex=${ i } data-name="js-${ name.toLowerCase() }">${ labels[name] || name }</button>`
                ).join('')
            }
            </div>
        </div>
    </div>`; 
}
```

Thanks

Paolo

**Comment by coderaiser** [comment]
Hi, it is an interesting idea :), we can add `buttons` property to `options` to every public method (`alert`, `confirm` and `prompt`) it can look like this for [prompt](https://github.com/coderaiser/smalltalk/blob/v3.0.0/lib/smalltalk.js#L18):

```js
exports.prompt = (title, msg, value = '', options) => {
    const type = getType(options);
    
    const val = String(value)
        .replace(/"/g, '&quot;');
    
    const valueStr = `<input type="${ type }" value="${ val }" data-name="js-input">`;
    const buttons = options.buttons || BUTTON_OK_CANCEL;
    return showDialog(title, msg, valueStr, buttons, options);
};
```

I think we should also add function `getButtons` similar to [getType](https://github.com/coderaiser/smalltalk/blob/v3.0.0/lib/smalltalk.js#L33) so we can use:

```js
const buttons = getButtons(options) || BUTTON_OK_CANCEL;
return showDialog(title, msg, valueStr, buttons, options);
``` 

Also tests should be added.

And ofcourse all this stuf would not work work [smalltalk.native](https://github.com/coderaiser/smalltalk/blob/v3.0.0/lib/smalltalk.native.js).

It is a great idea for a pull request :)

**Comment by paologf** [comment]
Hi @coderaiser I just modified the code, but I don't understand how to run test. Could you please explain me?

**Comment by coderaiser** [comment]
@paologf,  you can run `npm test` for this purpose.

**Comment by coderaiser** [comment]
Added in https://github.com/coderaiser/smalltalk/pull/15.
Landed in v[3.1.0](https://github.com/coderaiser/smalltalk/releases/tag/v3.1.0)
